modelling/src/test/java/org/axonframework/modelling/command/inspection/AnnotatedAggregateMetaModelFactoryTest.java

```diff
@@ -16,6 +16,7 @@
 
 package org.axonframework.modelling.command.inspection;
 
+import jakarta.persistence.Id;
```

Comment:
I expect a testcase to be added for an aggregate with the javax ID as well

Reply:
added one for javax id.